pegasus/migrations/106_add_popularityrank_to_tutorials.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :popularityrank, Integer, null: false
```

Comment:
If null values are not allowed, I would expect this migration to populate the column for existing rows.